tox.ini

```diff
@@ -15,3 +15,11 @@ commands =
         --cov=queuelib \
         --cov-report=term \
         {posargs:queuelib}
+
+[testenv:flake8]
+basepython = python3
+deps =
+    {[testenv]deps}
+    pytest-flake8
+commands =
+    pytest --flake8 .
```

Comment:
Asking out of sheer ignorance: what's the benefit of using this instead of just `flake8` here? Wouldn't it remove the need for the `conftest.py` file?

Reply:
I don’t recall why I did it the first time where I did it, but my guess is that it was about having a single test command run both tests and lint stuff. However, given how we use it and that we can run everything with a single `tox` command, I see no good point for `pytest-flake8`.